### PR TITLE
add readme to dataset_access folder and minor access notebook changes

### PR DESCRIPTION
**Pull request by amsnyder**

### CODE REVIEW

File: dataset_access/conus404_explore.ipynb [hunk 1]
```diff
@@ -327,9 +298,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "users-users-pangeo",
+   "display_name": "drb_estuary_salinity",
```
Comment (amsnyder) [inline dataset_access/conus404_explore.ipynb:301]:
```suggestion
   "display_name": "users-users-pangeo",
```

File: dataset_access/conus404_explore.ipynb [hunk 2]
```diff
@@ -327,9 +298,9 @@
...
    "language": "python",
-   "name": "conda-env-users-users-pangeo-py"
+   "name": "python3"
```
Comment (amsnyder) [inline dataset_access/conus404_explore.ipynb:303]:
```suggestion
   "name": "conda-env-users-users-pangeo-py"
```

File: dataset_access/conus404_explore.ipynb [hunk 3]
```diff
@@ -341,7 +312,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.15"
+   "version": "3.7.6 (default, Jan  8 2020, 20:23:39) [MSC v.1916 64 bit (AMD64)]"
```
Comment (amsnyder) [inline dataset_access/conus404_explore.ipynb:315]:
```suggestion
   "version": "3.9.15"
```

File: dataset_access/conus404_spatial_aggregation.ipynb [hunk 4]
```diff
@@ -958,9 +962,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "users-users-pangeo",
+   "display_name": "drb_estuary_salinity",
```
Comment (amsnyder) [inline dataset_access/conus404_spatial_aggregation.ipynb:965]:
```suggestion
   "display_name": "users-users-pangeo",
```

File: dataset_access/conus404_spatial_aggregation.ipynb [hunk 5]
```diff
@@ -958,9 +962,9 @@
...
    "language": "python",
-   "name": "conda-env-users-users-pangeo-py"
+   "name": "python3"
```
Comment (amsnyder) [inline dataset_access/conus404_spatial_aggregation.ipynb:967]:
```suggestion
   "name": "conda-env-users-users-pangeo-py"
```

File: dataset_access/conus404_spatial_aggregation.ipynb [hunk 6]
```diff
@@ -972,7 +976,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.15"
+   "version": "3.7.6 (default, Jan  8 2020, 20:23:39) [MSC v.1916 64 bit (AMD64)]"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "85ce4eb777a47250062e69db63a13ea88790b42e9bccd9f9c9a4eb5ac7d53107"
+   }
```
Comment (amsnyder) [inline dataset_access/conus404_spatial_aggregation.ipynb:984]:
```suggestion
   "version": "3.9.15"
```

File: dataset_access/conus404_explore.ipynb [hunk 7]
```diff
@@ -269,35 +269,6 @@
     "da.hvplot(x='time', grid=True)"
    ]
   },
-  {
```
Comment (amsnyder) [inline dataset_access/conus404_explore.ipynb:272]: I am suggseting we delete this, as it is covered in the temporal aggregation notebook. I noticed that you used `.mean()` here, and you use `.resample()` in the temporal aggregation notebook. Is one method better than the other?

Comment (rsignell-usgs) [reply dataset_access/conus404_explore.ipynb:272]: resample is better for time aggregation -- more flexible

**Review by rsignell-usgs — APPROVED** [review]

**Pull request merged**